types/parse/index.d.ts

```diff
@@ -458,9 +465,9 @@ declare namespace Parse {
      *   }
      * });</pre></p>
      */
-    class Query<T extends Object = Object> extends BaseObject {
+    class Query<T = Object, K extends Unobject<T> = Unobject<T>> extends BaseObject {
```

Comment:
Why removing `extends Object`?

```suggestion
    class Query<T extends Object = Object, K extends Unobject<T> = Unobject<T>> extends BaseObject {
```

Reply:
Good question, not sure what happened over there. I've reverted it.